### ISSUE TITLE
The progress bar can't roll while running validation and test

### ISSUE THREAD

**Issue by MooreManor** [Lightning-AI/lightning#14237]
## 🐛 Bug

The progress bar can roll while training, but for validation and testing, it just stops and runs in the background. And the expected cost time of validation and test can't show up. 

![image](https://user-images.githubusercontent.com/49632327/185029562-608faa20-1793-439f-9b71-0ff1d6fc09e0.png)

### To Reproduce
Just a normal procedure.

### Expected behavior

The progress bar can roll forward when running validation and testing. 

### Environment

* CUDA:
        - GPU:
                - NVIDIA GeForce RTX 3070 Laptop GPU
        - available:         True
        - version:           11.1
* Lightning:
        - pytorch-lightning: 1.5.0
        - torch:             1.8.0
        - torchaudio:        0.8.0
        - torchmetrics:      0.9.3
        - torchvision:       0.9.0
* Packages:
        - absl-py:           1.2.0
        - aiohttp:           3.8.1
        - aiosignal:         1.2.0
        - async-timeout:     4.0.2
        - asynctest:         0.13.0
        - attrs:             22.1.0
        - cachetools:        5.2.0
        - certifi:           2022.6.15
        - charset-normalizer: 2.1.0
        - colorama:          0.4.5
        - einops:            0.4.1
        - frozenlist:        1.3.1
        - fsspec:            2022.7.1
        - future:            0.18.2
        - google-auth:       2.10.0
        - google-auth-oauthlib: 0.4.6
        - grpcio:            1.48.0
        - idna:              3.3
        - importlib-metadata: 4.12.0
        - markdown:          3.4.1
        - markupsafe:        2.1.1
        - multidict:         6.0.2
        - numpy:             1.21.6
        - oauthlib:          3.2.0
        - packaging:         21.3
        - pillow:            9.2.0
        - pip:               22.1.2
        - protobuf:          3.19.4
        - pyasn1:            0.4.8
        - pyasn1-modules:    0.2.8
        - pydeprecate:       0.3.1
        - pyparsing:         3.0.9
        - pytorch-lightning: 1.5.0
        - pyyaml:            6.0
        - requests:          2.28.1
        - requests-oauthlib: 1.3.1
        - rsa:               4.9
        - setuptools:        61.2.0
        - six:               1.16.0
        - tensorboard:       2.10.0
        - tensorboard-data-server: 0.6.1
        - tensorboard-plugin-wit: 1.8.1
        - torch:             1.8.0
        - torchaudio:        0.8.0
        - torchmetrics:      0.9.3
        - torchvision:       0.9.0
        - tqdm:              4.64.0
        - typing-extensions: 4.3.0
        - urllib3:           1.26.11
        - werkzeug:          2.2.2
        - wheel:             0.37.1
        - wincertstore:      0.2
        - yarl:              1.8.1
        - zipp:              3.8.1
* System:
        - OS:                Windows
        - architecture:
                - 64bit
                - WindowsPE
        - processor:         AMD64 Family 25 Model 80 Stepping 0, AuthenticAMD
        - python:            3.7.13
        - version:           10.0.19041

cc @awaelchli @rohitgr7

**Comment by MooreManor** [comment]
And the validation starts from the middle of the training epoch. Shouldn't it start at the end of the training epoch?

**Comment by rohitgr7** [comment]
did you try it out with the latest version. Several fixes were made to it after v1.5 release

**Comment by MooreManor** [comment]
@rohitgr7 
I also tried v1.6 release, but it still didn't work.

**Comment by awaelchli** [comment]
The PyCharm console doesn't work well with tqdm. This has been a long issue we can't do much about. You may want to try the rich progress bar as an alternative. 

```py
Trainer(callbacks=[RichProgressBar])
```
